android-driver/src/main/scala/com/fortysevendeg/mvessel/api/impl/AndroidCursor.scala

```diff
@@ -1,12 +1,15 @@
 package com.fortysevendeg.mvessel.api.impl
 
-import android.database.Cursor
+import android.content.ContentResolver
+import android.database.{ContentObserver, CharArrayBuffer, DataSetObserver, Cursor}
+import android.net.Uri
+import android.os.Bundle
 import com.fortysevendeg.mvessel.api.CursorType.Value
 import com.fortysevendeg.mvessel.api.{CursorType, CursorProxy}
 
-class AndroidCursor(cursor: Cursor) extends CursorProxy {
+class AndroidCursor(cursor: Cursor) extends CursorProxy with Cursor {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Number of methods in class exceeds 30](https://www.codacy.com/app/47deg/mvessel/file/965563501/issues/source?bid=2442814&fileBranchId=2859700#l10)